code/modules/mob/living/carbon/human/human_movement.dm

```diff
@@ -69,12 +69,12 @@
 		if(dna.mutantrace == "slime")
 			tally *= 2
 		else
-			tally -= 10
+			tally -= 5
 
 	if(reagents.has_reagent("frostoil") && dna.mutantrace == "slime")
 		tally *= 5
 
-	return max((tally+config.human_delay), -1) //cap at -1 as the 'fastest'
+	return max((tally+config.human_delay), -100) //cap at -100 as the 'fastest'
```

Comment:
Whether this variable specifically affects the end result was not specifically tested, but was included to ensure it would not pose problems